app/user_task.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint
+from app.models import UserTask
+
+
+user_task_blueprint = Blueprint("user_task", __name__)
+
+
+@user_task_blueprint.route("/task/<id>/user_task", \
+                           methods=("GET",))
+def get_task(identifier):
+    user_task: UserTask = UserTask.query.filter(
```

Comment:
You can also use `<Some model>.query.get()` to get something by its primary key!
```suggestion
    user_task: UserTask = UserTask.query.get(identifier)
```